services/sidecar/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM python:3.6-alpine as base
+ARG PYTHON_VERSION="3.6"
```

Comment:
freeze version, i.e. 3.6.10

Reply:
well I was wondering whether or not... I'll go for your solution.